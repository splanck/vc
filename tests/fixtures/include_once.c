#include "once.h"
#include "once.h"
int main() { return 0; }
