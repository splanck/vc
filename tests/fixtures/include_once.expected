int once_var;
int main() { return 0; }
